libs/feature/editor/src/lib/fields.config.ts

```diff
@@ -83,14 +83,25 @@ export const RECORD_ABSTRACT_FIELD: EditorField = {
   },
 }
 
+export const RECORD_GRAPHICAL_OVERVIEW_FIELD: EditorField = {
+  model: 'overviews',
+  formFieldConfig: {
+    labelKey: marker('editor.record.form.field.overviews'),
```

Comment:
This will need a translation.

Reply:
This had a translation but should have been lost with a rebasing ;/